rest-api/tools/csv_importer.py

```diff
@@ -0,0 +1,98 @@
+"""Imports entities into the database based on a CSV file.
```

Comment:
FYI: I'm ignoring all these files (assuming you haven't rebased) so let me know if there are some changes

Reply:
Yeah, haven't changed them.